bp_common/test/src/demos/Makefile

```diff
@@ -1,8 +1,8 @@
 
 include Makefile.frag
 
```

Comment:
So this means we're using bsg_newlib right?

Reply:
I think it works with either standard or bsg_newlib. We need that to import some printf functionality from the C library.